bin/update-package.sh

```diff
@@ -63,9 +63,14 @@ for package in packages/*; do
 	# Before we commit any changes, ensure that the repo has the basics we need for any package.
 	if [ ! -f "composer.json" -o ! -d "src" ]; then
 		echo "  Those changes remove essential parts of the package. They will not be committed."
-	# Commit if there is any change that could be committed
-	elif [ -n "$(git status --porcelain)" ]; then
+		continue
+	fi
 
+	# Let's make sure composer.json is valid.
+	$( composer validate -n >/dev/null 2>&1 ) || continue
```

Comment:
@zinigor I can't get this to work properly.

When validation fails, it stops the script altogether instead of going to the next loop iteration and the next package. I'm probably missing something obvious here. Can you tell what?

Reply:
Related discussion: p1584094349206300-slack-jetpack-crew